test/e2e/storage/csi_mock_volume.go

```diff
@@ -24,7 +24,7 @@ import (
 	"strings"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	storage "k8s.io/api/storage/v1"
```

Comment:
why this change btw?

Reply:
i think that's my editor automatically running "go fmt". i'm on go version 1.11.1